src/main.go

```diff
@@ -3,12 +3,12 @@ package main
 import (
 	"log"
 	"net/http"
+	"src/services/palindrome"
 
-	"github.com/oreuta/elementary-service/src/services/squarert"
 )
 
 func main() {
 	log.Println("Elementary-service started...")
-	http.HandleFunc("/squarert", squarert.Handler)
+	http.HandleFunc("/squarert", palindrome.Handler)
```

Comment:
Oops, line 12 (http.HandleFunc("/squarert", palindrome.Handler)  should be http.HandleFunc("/palindrome", palindrome.Handler). i'll fix it soon